polypomo

```diff
@@ -133,6 +138,57 @@ class Status:
             self.timer = Timer(self.worktime)
 
 
+@contextmanager
+def setup_conn():
+    global DATABASE_PATH
+    # check if the database exist
+    if (os.path.exists(DATABASE_PATH)):
+        print('database already exist')
+        conn = sqlite3.connect('polypomoDB.db')
+        cur = conn.cursor()
+        try:
+            yield cur
+        finally:
+            conn.commit()
+            conn.close()
+    else:
+        print('initiate database')
+        conn = sqlite3.connect('polypomoDB.db')
+        cur = conn.cursor()
+        cur.execute('''CREATE TABLE data
```

Comment:
Most of the original code uses strings with (`"` - double quotes).
Could you use that throughout your code?

The present repository isn't configured to validate syntax and style but I'll seek to that after this.